cocos2d/CCTextureCache.m

```diff
@@ -238,7 +242,7 @@ -(void) addImageAsync: (NSString*)path target:(id)target selector:(SEL)selector
 //            [EAGLContext setCurrentContext:nil];
```

Comment:
i would gratuitously kill all these comments instead of just updating the director referencing - the previous code exists in version control should we need to reference it